modules/restful_token_auth/plugins/restful/restful_token_auth/token_auth/access_token/1.0/access_token__1_0.inc

```diff
@@ -11,6 +11,7 @@ $plugin = array(
   'authentication_types' => array(
     'cookie',
     'basic_auth',
+    'token',
```

Comment:
Adding token auth here to you can delete you access token while authenticated with your access token.